### PR TITLE
node js week3 warmup home work added

### PR DESCRIPTION
**Pull request by seenachacko**

### CODE REVIEW

File: nodejs/week3/homework/warmup/index.js [hunk 1]
```diff
@@ -0,0 +1,75 @@
+const express = require("express");
+const app = express();
+
```
Comment (cecastosic) [inline nodejs/week3/homework/warmup/index.js:10]: I think you are missing here 
```suggestion
app.use(
  express.urlencoded({
    extended: true,
  })
);
app.use(express.json());
```
Have you tested POST requests with Postman?

File: nodejs/week3/homework/warmup/index.js [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+    if(isNaN( multipliedResult) ){
+       res.status(400).send("values should be integer")
+
+    }else{
+        res.send(`Result :${ multipliedResult}`)
+    }
+
+});
+/*
+app.post("/multiply", (req, res) =>{
```
Comment (cecastosic) [inline nodejs/week3/homework/warmup/index.js]: remove this part if it is not used
and remove all console.log(...)

File: nodejs/week3/homework/warmup/index.js [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+
+app.get("/", (req, res) => res.send("nodejs week3 homework"));
+
+app.listen(3000, () => console.log(`Calculator:listening on port 3000`));
+app.get("/calculator/multiply", (req, res) => {
+    const values = Object.values(req.query).flat()
+    console.log(values);
+    const multiply = values.reduce((acc, val) => acc * val)
+    
+    if(isNaN(multiply)){
```
Comment (cecastosic) [inline nodejs/week3/homework/warmup/index.js]: Nice that you thought about if the query parameters were numbers 💪

**Review by cecastosic — COMMENTED** [review]
Good job 💪 
- Please format files with Prettier https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode
- Some part of the code is repeating, any ideas on how to optimize it? 🙂 
- Can you please add POST requests for addition, subtraction and division, and test it with Postman for example? Thanks

File: nodejs/week3/homework/warmup/index.js (hunk 3 above)
Comment (seenachacko) [reply nodejs/week3/homework/warmup/index.js]: Thanks for your valuable feedbacks

**Pull request merged**